mewarpx/mewarpx/mwxrun.py

```diff
@@ -68,20 +64,54 @@ def init_grid(self, lower_bound, upper_bound, number_of_cells, **kwargs):
                 of list has to equal number of dimensions.
             upper_bound (list): Maximum coordinates for all direction; length
                 of list has to equal number of dimensions.
-            cell_nums (list): Number of grid cells in each direction.
+            number_of_cells (list): Number of grid cells in each direction.
             kwargs (dict): Dictionary containing extra arguments. These can
                 include:
+                - ``use_rz`` to set the simulation to use cylindrical
+                  coordinates.
                 - boundary condition settings in the form
                   ``bc_fields_x/y/z/r_min/max`` (can be 'periodic', 'dirichlet'
                   or 'none') or ``bc_particles_x/y/z/r_min/max`` (can be
                   'periodic', 'absorbing' or 'reflecting').
                 - the minimum number of tiles via ``min_tiles``.
```

Comment:
While we're changing the docstrings, I think we should start with the key word rather than the description of the argument. ex.
```
- bc_type_dim : specify boundary conditions for the desired boundary. ``bc_fields_x/y/z/r_min/max`` (can be 'periodic', 'dirichlet' or  'none') or ``bc_particles_x/y/z/r_min/max`` (can be 'periodic', 'absorbing' or 'reflecting').
- ``min_tiles`` : the minimum number of tiles 
```